docs/howto/krustlet-on-microk8s.md

```diff
@@ -0,0 +1,141 @@
+# Running Krustlet on [microk8s](https://microk8s.io)
+
+These are steps for running krustlet node(s) and [microk8s](https://microk8s.io) on the same machine.
```

Comment:
Krustlet should be capitalized unless referring to the binary (which should be wrapped in backticks). Also, I think from the website that MicroK8s is written with the M and K capitalized unless I am missing something. This applies across the document. (And yes, I figured you copy/pasted this, so I went back and fixed it in the other docs that I got this wrong in 😆)

Reply:
Fixed: ee852270c8e6386c95b618ff365dad7e3bf4d328